doc/specifications/proposed/field_type.md

```diff
@@ -0,0 +1,308 @@
+Field Type Creation Simplification
+==================================
+
+Overview
+--------
+
+First provide a couple of default implementations on the
+`eZ\Publish\Core\FieldType\FieldType` class. Omit all optional methods
+from the
+[tutorial](https://doc.ez.no/display/EZP/eZ+Publish+5+Field+Type+Tutorial).
```

Comment:
If we omit optional methods, there will be no easy reference on how to implement them. Maybe we should create at least one "advanced" implementation that shows optional methods?

Side note: current version of FT tutorial is [here](https://github.com/ezsystems/TweetFieldTypeBundle/blob/documentation/docs/index.md)